Shaders/DisplayDepth.fx

```diff
@@ -52,36 +42,30 @@ uniform int iUIUpsideDown <
 	ui_category = "Preprocessor";
 	ui_type = "combo";
 	ui_label = "Upside Down";
-	ui_tooltip = "On:  RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=1\n"
-	             "Off: RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=0";
-	ui_items = "Off\0On\0";
+	ui_items = "RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=0\0RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=1\0";
 > = __DISPLAYDEPTH_UI_UPSIDE_DOWN_DEFAULT__;
 
 uniform int iUIReversed <
 	ui_category = "Preprocessor";
 	ui_type = "combo";
 	ui_label = "Reversed";
-	ui_tooltip = "On:  RESHADE_DEPTH_INPUT_IS_REVERSED=1\n"
-	             "Off: RESHADE_DEPTH_INPUT_IS_REVERSED=0";
-	ui_items = "Off\0On\0";
+	ui_items = "RESHADE_DEPTH_INPUT_IS_REVERSED=0\0RESHADE_DEPTH_INPUT_IS_REVERSED=1\0";
 > = __DISPLAYDEPTH_UI_REVERSED_DEFAULT__;
 
 uniform int iUILogarithmic <
 	ui_category = "Preprocessor";
 	ui_type = "combo";
 	ui_label = "Logarithmic";
-	ui_tooltip = "On:  RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=1\n"
-	             "Off: RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=0";
-	ui_items = "Off\0On\0";
+	ui_items = "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=0\0RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=1\0";
+	ui_tooltip = "Change this setting if far away objects have stripes in them";
 > = __DISPLAYDEPTH_UI_LOGARITHMIC_DEFAULT__;
 
-uniform bool bUIShowNormals <
+uniform int iUIShowNormals <
 	ui_category = "Debug";
 	ui_type = "combo";
-	ui_label = "Show Normals";
-	ui_tooltip = "On:  Show normals\n"
-	             "Off: Show depth";
-> = true;
+	ui_label = "Debug";
```

Comment:
The label of the debugger option to be added in the future will be "Debug"

Reply:
I've changed the uniforms for the preprocessor settings from On/Off to have the actual values as items - which is more meaningful for the user I think. And to be consistent I've also done this to the debug option. Maybe it is better to change the category to "Options" or "Settings" and the label to "Output" (or something like that). Changing between normals and depth is definitely not debugging ;-)

Reply:
about category, i have no ideas.
about label, i am thinking too.
![present_type](https://user-images.githubusercontent.com/42232001/48813763-869d0d80-ed7b-11e8-98f0-8beb6939a56e.png)